src/iarc7_sensors/landing_detector.py

```diff
@@ -74,6 +79,14 @@ def odom_callback(msg):
                 rospy.logwarn_throttle(1.0,
                     'Landing Detector is not receiving switch messages fast enough')
 
+        # If this fails something is seriously wrong, so we're ok with crashing
```

Comment:
This is not ok because then fc_comms will not be able to execute a safety landing.

Reply:
No this shouldn't fail unless there's memory corruption in the node or something, or we haven't received a transform for so long that the internal buffer in tf is empty, and I'm not even sure if that's possible